### PR TITLE
Added Tower Of Hanoi in PHP

### LINKED ISSUE #2479: Tower of Hanoi
- [x] C
- [x] C++
- [x] C# - @biekymahato
- [ ] CoffeeScript
- [x] Dart - @disha2sinha
- [x] Go - @rbjoshi1309
- [x] Java
- [x] JS
- [x] Kotlin - @chandbud5
- [x] Php - @chandrikadeb7
- [x] Python
- [x] Readme
- [x] Ruby - @HardevKhandhar
- [ ] Rust
- [x] TypeScript - @vyom153069

### PR DESCRIPTION
**Pull request by chandrikadeb7**
Fixes #2479 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added PHP implementation of Tower Of Hanoi
- 

#### Languages Used:
- PHP
- 

#### Files Added:
- Tower_Of_Hanoi/Tower_Of_Hanoi.php
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by chandrikadeb7** [comment]
@raghav-dalmia Please review.

**Comment by chandrikadeb7** [comment]
@MastersAbh Pls review.

File: Tower_Of_Hanoi/Tower_Of_Hanoi.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?php
```
Comment (singh-shreya6) [inline Tower_Of_Hanoi/Tower_Of_Hanoi.php]: Please add a brief problem description on top as multi line comment.

**Comment by chandrikadeb7** [comment]
@singh-shreya6  Please assign the respective labels to my PR

**Comment by chandrikadeb7** [comment]
@jainaman224 Please assign the labels for GSSoC. My score was not updated hence.

**Pull request merged**